docs/src/Modal.doc.js

```diff
@@ -177,6 +181,49 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="closeOnOutsideClickExample"
+    name="Prevent close on outside click"
+    description={`
+      Sometimes we need the user to complete some required action at a Modal.
+      We can ensure that by preventing the user closing the modal.
+      For that you can set: \`closeOnOutsideClick\` to \`false\`.
+    `}
+    defaultCode={`
+function Example(props) {
+  const [showModal, setShowModal] = React.useState(false);
+  return (
+    <>
```

Comment:
fyi no longer wrapped this in a `Box` with negative margin left. We only did that in the other examples because there were multiple buttons.